hr_worked_days_hourly_rate/models/hr_payslip_worked_days.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding:utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 Savoir-faire Linux. All Rights Reserved.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import fields, models
+
+
+class HrPayslipWorkedDays(models.Model):
+    _inherit = 'hr.payslip.worked_days'
+
+    def _compute_total(self):
+        self.total = self.number_of_hours * self.hourly_rate * self.rate / 100
+
+    hourly_rate = fields.Float(
+        'Hourly Rate',
+        default=0,
+        help="""
+The employee's standard hourly rate for one hour of work.
+Example, 25 Euros per hour.
+        """)
+    rate = fields.Float(
+        'Rate (%)',
+        default=100,
+        help="""
```

Comment:
Putting triple quota you introduce break lines on the translatable texts, making them ugly. Use instead:
```
help="A line very long that "
     "needs two lines"
```